app/src/main/java/com/android/wardrobeManager/StartActivity.java

```diff
@@ -1,26 +1,133 @@
+        displayShoes = (Shoes) randomClosetItem(Shoes.class);
+
+        shirtImage = findViewById(R.id.shirtImage);
+        shortsImage = findViewById(R.id.shortsImage);
+        shoesImage = findViewById(R.id.shoesImage);
+        updateColors();
     }
 
     public void goToCloset(View view) {
-        startActivity(new Intent(StartActivity.this, ClosetActivity.class));
+
+        Intent intent = new Intent(StartActivity.this, ClosetActivity.class);
+        intent.putExtra("closetClothes", closetClothes);
+        startActivity(intent);
     }
 
     public void goToExpandCloset(View view) {
-        startActivity(new Intent(StartActivity.this, ExpandClosetActivity.class));
+        Intent intent = new Intent(StartActivity.this, ExpandClosetActivity.class);
+        intent.putExtra("closetClothes", closetClothes);
+        intent.putExtra("previousActivity", "StartActivity");
+        startActivity(intent);
+    }
+
+    private void initializingClosetClothes() {
+
+        closetClothes = new ArrayList<>();
+        closetClothes.add(new Shirt(getResources().getColor(R.color.red)));
+        closetClothes.add(new Shirt(getResources().getColor(R.color.green)));
+        closetClothes.add(new Shoes(getResources().getColor(R.color.black)));
+        closetClothes.add(new Shorts(getResources().getColor(R.color.black)));
+        closetClothes.add(new Shorts(getResources().getColor(R.color.green)));
+        closetClothes.add(new Shoes(getResources().getColor(R.color.blue)));
+        closetClothes.add(new Shirt(getResources().getColor(R.color.pink)));
+        closetClothes.add(new Shirt(getResources().getColor(R.color.blue)));
+        closetClothes.add(new Shirt(getResources().getColor(R.color.blue)));
+        closetClothes.add(new Shirt(getResources().getColor(R.color.brown)));
+
+    }
+
+    public void shuffleShirt(View view) {
+        displayShirt = (Shirt) randomClosetItem(Shirt.class);
+        updateColors();
+    }
+
+    public void shuffleShorts(View view) {
+        displayShorts = (Shorts) randomClosetItem(Shorts.class);
+        updateColors();
+    }
+
+    public void shuffleShoes(View view) {
+        displayShoes = (Shoes) randomClosetItem(Shoes.class);
+        updateColors();
+    }
+
+    public void shuffleOutfit(View view) {
+        displayShirt = (Shirt) randomClosetItem(Shirt.class);
+        displayShorts = (Shorts) randomClosetItem(Shorts.class);
+        displayShoes = (Shoes) randomClosetItem(Shoes.class);
+        updateColors();
+    }
+
+    public ClothingItem randomClosetItem(Class clothingType) {
+        ArrayList<ClothingItem> items = new ArrayList<>();
+
+        for (int i = 0; i < closetClothes.size(); i++) {
+            if (closetClothes.get(i).getClass().equals(clothingType)) {
+                items.add(closetClothes.get(i));
+            }
+        }
+
+        if (items.size() > 0) {
+            return items.get((int) (Math.random() * items.size()));
+        }
+
+        return null;
```

Comment:
You are also probably going to want to make sure this doesn't return the previous clothing item that was displayed so the user gets a different one every time. Or you could just iterate to the next clothing item of that type in the list so it goes in the same order every time. When you get to the end of the list, you can just shuffle the clothing list. It is a quick and dirty solution that will be mostly right for now :). But leave it like this if you think it is better for now though.